conf/modules/annotate.config

```
@@ -18,7 +18,7 @@ process {
     // SNPEFF
     if (params.tools && (params.tools.split(',').contains('snpeff') || params.tools.split(',').contains('merge'))) {
         withName: 'SNPEFF_SNPEFF' {
-            ext.prefix       = { "${vcf.baseName.minus(".vcf")}_snpEff" }
+            ext.prefix       = { vcf.name - ".vcf" + "_snpEff" }
```

I don't think this needs changing
```suggestion
            ext.prefix       = { "${vcf.baseName.minus(".vcf")}_snpEff" }
```

---

How about?
```suggestion
            ext.prefix       = { vcf.baseName - ".vcf" + "_snpEff" }
```